internal/backend/db/dbgorm/sessions.go

```diff
@@ -176,6 +176,7 @@ func (db *gormdb) listSessions(ctx context.Context, f sdkservices.ListSessionsFi
 	if err := q.
 		Order(clause.OrderByColumn{Column: clause.Column{Name: "created_at"}, Desc: true}).
 		Order(clause.OrderByColumn{Column: clause.Column{Name: "session_id"}, Desc: true}).
+		Omit("inputs", "compressed_inputs").
```

Comment:
same as events, no need for compressed data
just get the inputs when really required (even on get you don't really need them unless specifically set)